app/main.py

```diff
@@ -1 +1,50 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class SkillMixin:
```

Comment:
why do you need this mixin if you have SoftwareEngineer and can save to_add_skills there?

Reply:
delete pls